Core/scripts/dirac-consumer.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+"""  This is a script to launch DIRAC consumers
+"""
+
+import sys
+from DIRAC.ConfigurationSystem.Client.LocalConfiguration import LocalConfiguration
+from DIRAC import gLogger
+from DIRAC.Core.Base.ConsumerReactor import ConsumerReactor
+
+def main():
+  """It launches DIRAC consumers
+  """
+  localCfg = LocalConfiguration()
+  consumerModuleNames = localCfg.getPositionalArguments()
+  if len( consumerModuleNames ) == 0:
```

Comment:
Yes. It was copy-paste from some agent code.
I changed the code to deal with one consumer module, and also fix it.